### ISSUE TITLE
Sync attempts after shutdown signal

### ISSUE THREAD

**Issue by xaur** [decred/dcrd#1740]
Not sure if it's a bug, but just in case.

```
[INF] BMGR: Processed <count> blocks (...)
[INF] BMGR: New valid peer <peer1> (outbound) (/dcrwire:0.3.0/dcrd:1.4.0/)
[INF] BMGR: New valid peer <peer2> (outbound) (/dcrwire:0.3.0/dcrd:1.4.0/)
[INF] BMGR: New valid peer <peer3> (outbound) (/dcrwire:0.3.0/dcrd:1.4.0/)
^C
[INF] DCRD: Received signal (interrupt).  Shutting down...
[INF] DCRD: Gracefully shutting down the server...
[WRN] SRVR: Server shutting down
[WRN] RPCS: RPC server shutting down
[INF] RPCS: RPC server shutdown complete
[INF] BMGR: Lost peer <peer4> (outbound)
[INF] BMGR: Syncing to block height <height> from peer <peer5>
[INF] BMGR: Lost peer <peer5> (outbound)
[INF] BMGR: Syncing to block height <height> from peer <peer6>
[INF] BMGR: Lost peer <peer6> (outbound)
[INF] BMGR: Syncing to block height <height> from peer <peer1>
[INF] BMGR: Lost peer <peer1> (outbound)
[INF] BMGR: Syncing to block height <height> from peer <peer2>
[INF] BMGR: Block manager shutting down
[INF] BMGR: Lost peer <peer2> (outbound)
[INF] BMGR: Syncing to block height <height> from peer <peer3>
[INF] AMGR: Address manager shutting down
[INF] SRVR: Server shutdown complete
[INF] DCRD: Gracefully shutting down the database...
[INF] DCRD: Shutdown complete
```

edit: v1.4.0 release

**Comment by davecgh** [comment]
Thanks for reporting.

I'll leave this open as it can be addressed when we eventually get around to reworking the sync logic, but this really isn't an issue.  The block manager doesn't shutdown until after all of the peers are shutdown (which necessarily disconnects them), so as the current sync peer is shutdown/disconnected, it chooses another one and logs the selection.  Any attempts to send any messages to peers are ignored during shutdown, so it's really just a case of the log message being a bit misleading.

**Comment by davecgh** [comment]
This was resolved by #2555.
